### PR TITLE
Handle other  errors - other than 404

### LINKED ISSUE #379: Frontend: Default error page for failed backend requests
* **I'm submitting a ...**
  * feature request

* **What is the current behavior?**
I see a loading spinner if the backend responds with 500.

* **What is the expected behavior?**
I should see an error page "Hm, something went wrong."

* **What is the motivation / use case for changing the behavior?**
Better user experience. Tell the user we have technical difficulties on our side.
 
* **If you want to request a whole new feature, please provide a user story (otherwise delete):**
  * As a `user`
  * I want `to see an error page if there any backend requests failed`
  * So that `I know what's going on and don't think the page is still loading`

* **What could be a possible solution? (not obligatory)**
Create a default error page in the top folder `/frontend/app/templates/`.

### PR DESCRIPTION
**Pull request by sigu**
closes #379
## Done
- use error template that is rendered incase error is not caught by `404`

## Screenshot
screenshot-2017-10-30 rundfunk-mitbestimmen

## GIF
peek 2017-10-30 17-55

### CODE REVIEW

File: frontend/tests/acceptance/error-page-test.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  mockFindRecord
+} from 'ember-data-factory-guy';
+import { authenticateSession } from 'frontend/tests/helpers/ember-simple-auth';
+
+describe('Acceptance | Errors', function() {
+  let application, broadcastMock;
+  beforeEach(function() {
+    application = startApp();
+    mockSetup();
+    broadcastMock = mockFindRecord('broadcast');
```
Comment (sigu) [inline frontend/tests/acceptance/error-page-test.js]: @roschaefer not sure if I really need this, I declared it so I can fake a `500` response as shown in line [32](https://github.com/roschaefer/rundfunk-mitbestimmen/pull/385/files#diff-2699258f09184654585e148488e77c80R32 )

File: frontend/tests/acceptance/error-page-test.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      authenticateSession(application, sessionData);
+    });
+
+    describe('HTTP 500 from backend', function() {
+      beforeEach(function() {
+        broadcastMock.fails({ status: 500 });
+      });
+
+      it('shows an error page', function() {
+        visit('/invoice');
```
Comment (sigu) [inline frontend/tests/acceptance/error-page-test.js]: This is what is currently giving the error, I cannot get the http status from the log. It does not send a backend request too.
Good thing, it is caught by the error page

Comment (roschaefer) [reply frontend/tests/acceptance/error-page-test.js]: That's interesting. I learned from the maintainer of ember-data-factory-guy that `mockFindRecord` only captures the request to `/broadcast/:id` with the correct `:id`. You could use `mockFindAll('broadcast')` and visit `find-broadcasts` that should give the 500 error for sure.

File: frontend/translations/de.yaml [hunk 3]
```diff
@@ -587,3 +587,7 @@ about-us:
 404:
   header: Ups..
   sub-header: Wir konnten die Seite nicht finden.
+
+error:
+  header: Ups..
+  sub-header: Hm, etwas ist schief gelaufen
```
Comment (roschaefer) [inline frontend/translations/de.yaml]: that's perfect German :de:

File: frontend/tests/acceptance/error-page-test.js (hunk 1 above)
Comment (roschaefer) [reply frontend/tests/acceptance/error-page-test.js]: you did it right :+1:

File: frontend/translations/de.yaml (hunk 3 above)
Comment (sigu) [reply frontend/translations/de.yaml]: Google translate 😛

File: frontend/tests/acceptance/error-page-test.js (hunk 2 above)
Comment (sigu) [reply frontend/tests/acceptance/error-page-test.js]: Now that we have the object we can simulate a 500 error from the backend

**Comment by roschaefer** [comment]
@sigu thanks!

**Pull request merged**